_posts/2021-03-01-whats-next-for-uswds.md

```diff
@@ -0,0 +1,334 @@
+---
+title: "What’s next for the U.S. Web Design System"
+tags:
+- updates
+category: About
+author: Arden Klemmer, Natalie Kurz, Norah Maki, and Kim Schulke
+meta:
+  og:image: /img/2021-next-blog/post-2-social-share.jpg
+excerpt: "Today we’re releasing a new and exciting report and strategy that outlines what’s next for the U.S. Web Design System (USWDS) and our community."
+---
+
+Today we’re releasing a new and exciting report and strategy that outlines [what’s next for the U.S. Web Design System (USWDS)]({{ site.baseurl }}/next/) and our community.
+
+Thanks to the participation of content managers, designers, engineers, policy analysts, and thought leaders across dozens of federal agencies, we conducted the largest research effort in the history of the design system. We conducted 60 interviews across 33 teams to better understand how teams are dealing with **emergency response** situations (specifically COVID-19) in terms of their digital communications, as well as their needs as users of the design system.
+  
+Taking action on what [we learned from the research](#), we created and updated several design system components and templates to address the common needs agencies faced.  
```

Comment:
Before this gets published, please update this link with the final permalink for the previous blog post!